src/annotator/components/toolbar.js

```diff
@@ -1,6 +1,7 @@
 import { SvgIcon } from '@hypothesis/frontend-shared';
 import propTypes from 'prop-types';
-import { createElement } from 'preact';
+import { createElement, render } from 'preact';
```

Comment:
render isn't a needed import anymore, is it? Same for Doodle, as we're using it through our DoodleController